site/hxapi/hotel/av/index.md

```diff
@@ -48,6 +48,7 @@ NB: All parameter names are case sensitive.
  | ParkingDays       | Integer  | [0-9] 2 chars | Y        | NB: The maximum duration accepted for ParkingDays is 30.|
  | System      | String  | [A-Z] 3 chars | Y*       | For European products, you need to pass in the value of `System=ABG` (the default is `System=ABC`, which is UK products only). |
  | lang        | String  | [A-Z] 2 chars | Y*       | Required for requests for European products. (Values available are `en`, `de`, `it`, `es`, `pt` and `nl`.)|
+ | Terminal | String | [A-Z1-9] 1 char | N | An optional field when requesting hotel and parking that will filter the availble results to those that serve the given terminal. Terminal options can be found using the [Terminal](/hxapi/terminal) request. |
```

Comment:
This says 1 char but the terminal response can give you results in a different format, e.g. https://api.holidayextras.co.uk/sandbox/v1/terminal/LGW?key=YourKey will return `North` and `South`